### DISCUSSION TITLE
Get access to the improved version of ChatGPT Writer 🔥

### DISCUSSION THREAD

**Discussion by GorvGoyl** [GorvGoyl/ChatGPT-Writer#35] [Provide Feedback and Ideas]
I’m building a much better and more personalized version of **ChatGPT Writer** ([chatgptwriter.ai](https://chatgptwriter.ai/)) for you.

- No more _Cloudflare security check_ connection issues.
- Highly personalised results.
- New features like email tones, one-click responses, etc.
- Support for more languages.

Please fill out this short form to get access to it: https://docs.google.com/forms/d/e/1FAIpQLSeLLFQTOSkBF6VAHCQDTAmXU-j8f-VX810EWTuSPJFbd0OPKQ/viewform?usp=sf_link

**Comment by tiagozip** [comment]
Cool, form filled!
